spec/spec_helper.rb

```diff
@@ -14,6 +14,14 @@
   config.filter_run :focus
   config.order = "random"
 
+  config.before(:example) do
+    implementation = ENV["SASS_IMPLEMENTATION"]&.to_sym
+    unless implementation.nil?
```

Comment:
Why not `if implementation` directly..?

Reply:
More explicit in terms of checking empty string (defined) vs nil (undefined). Although I guess we can also treat empty string same as undefined.

Reply:
Never mind, I came from other languages and I did not know that in ruby empty string or empty symbol is truthy. 🙃